app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -110,7 +110,7 @@
     private Display.Mode[] mDisplayModes;
     private boolean refreshRateSwitchingEnabled;
 
-    public PlaybackController(List<BaseItemDto> items, IPlaybackOverlayFragment fragment) {
+    public PlaybackController(List<BaseItemDto> items, @Nullable IPlaybackOverlayFragment fragment) {
```

Comment:
Why did you add Nullable for the second parameter but did nothing for the first parameter?

Reply:
I reverted this one, because all calls to it don't pass null and shouldn't probably.